### PR TITLE
build(docker): Start building Python 3 images

### PR DESCRIPTION
**Pull request by BYK**

### CODE REVIEW

File: docker/cloudbuild.yaml [hunk 1]
```diff
@@ -8,20 +9,38 @@ steps:
...
   timeout: 300s
-# Smoke tests
+- name: 'gcr.io/kaniko-project/executor:v0.22.0'
+  id: runtime-image-py3
+  waitFor:
+    - builder-run
+  args: [
+    '--cache=true',
+    '--build-arg', 'SOURCE_COMMIT=$COMMIT_SHA',
+    '--build-arg', 'PY_VER=3.6',
```
Comment (joshuarli) [inline docker/cloudbuild.yaml:35]: You should match what we have in `.python-version` here.

```suggestion
    '--build-arg', 'PY_VER=3.6.10',
```

**Review by evanpurkhiser — APPROVED** [review]
If it passes lg2m

File: docker/cloudbuild.yaml (hunk 1 above)
Comment (BYK) [reply docker/cloudbuild.yaml:35]: Why are we not using the latest patch version?

Comment (mattrobenolt) [reply docker/cloudbuild.yaml:35]: +1 here, let's keep this strictly pinned to match. We can just bump the patch release, but I don't fully trust docker python images without strict pinning. Things like `pip` will get upgraded or whatever in a new release that is opaque to us until it breaks, etc.

File: src/sentry/runner/importer.py [hunk 2]
```diff
@@ -17,7 +19,7 @@ def show(self, file=None):
             from click._compat import get_text_stderr
 
             file = get_text_stderr()
-        click.secho("!! Configuration error: %s" % self.format_message(), file=file, fg="red")
+        click.secho("!! Configuration error: %s" % str(self), file=file, fg="red")
```
Comment (wedamija) [inline src/sentry/runner/importer.py:22]: `six.text_type`

**Pull request merged**